tasks/main.yaml

```diff
@@ -13,10 +15,25 @@
     baseurl: "{{ rabbitmq_epel_yumrepo_url }}"
     gpgkey: "{{ rabbitmq_epel_yumrepo_gpgkey }}"
     params: "{{ rabbitmq_epel_yumrepo_params }}"
-  when: rabbitmq_epel_install == true
+  when: >
+    ansible_os_family == 'RedHat'
```

Comment:
Here you forgot the original variable check. Please change it to this:

```
when: >
  ansible_os_family == 'RedHat' and
  rabbitmq_epel_install == true
```

Reply:
Ah whoops, you are correct it's obvious when looking at the diff